types/node/http.d.ts

```diff
@@ -113,7 +113,86 @@ declare module 'http' {
     }
     // outgoing headers allows numbers (as they are converted internally to strings)
     type OutgoingHttpHeader = number | string | string[];
-    interface OutgoingHttpHeaders extends NodeJS.Dict<OutgoingHttpHeader> {}
+    interface OutgoingHttpHeaders extends NodeJS.Dict<OutgoingHttpHeader> {
+        accept?: string | undefined;
+        'accept-charset'?: string | undefined;
```

Comment:
`string[]` can be used almost with all headers

Reply:
Is the same applied to ```IncomingHeaders```? If so, I propose we match them, because as of right now, most incoming headers accept only string.

Reply:
See https://github.com/nodejs/node/blob/31e727db7e0f57bd71d35ce8c1cf378bab309b0f/test/parallel/test-http-server-multiheaders2.js#L32-L45

Reply:
> Is the same applied to IncomingHeaders?

Not sure, they can be received as joined string. Need testing

Reply:
Looks like only `string` for `IncomingHeaders`: https://github.com/nodejs/node/blob/31e727db7e0f57bd71d35ce8c1cf378bab309b0f/test/parallel/test-http-multiple-headers.js#L23-L29